packages/perseus/src/util/graphie.ts

```diff
@@ -1411,6 +1434,57 @@ const labelDirections = {
     "above left": [-1.0, -1.0],
 } as const;
 
+// A range for a Graphie can be specified in one of three forms:
```

Comment:
I wonder if it's worth starting to pull out these types of things into their own files.  There are other concepts of `Range` that would be nice to start unifying:

```sh
rg "type Range"
packages/perseus/src/perseus-types.ts
6:export type Range = [number, number];

packages/perseus-editor/src/widgets/number-line-editor.tsx
11:type Range = [number, number];

packages/perseus-editor/src/widgets/interactive-graph-editor.tsx
46:type Range = [number, number]; // [min, max]

packages/perseus/src/util/geometry.ts
15:export type Range = [number, number];
```

I am good if you don't want to get distracted with this refactoring in this PR or just push it off as low priority too.

Reply:
In general, duplicated types don't worry me much, because:

- they have no runtime cost
- TypeScript is structurally typed, so all these types are interchangeable
- it's nice to be able to see the definition of a commonly-used type without jumping around between files, so there is an ergonomic advantage to having types (re)defined close to where they're used.
- type checking should fail if the various definitions of a type get out of sync in a way that matters (I say *should* because our type coverage currently isn't great).